draft/2025-03-26-this-week-in-rust.md

```diff
@@ -244,22 +232,20 @@ Rusty Events between 2025-03-26 - 2025-04-23 🦀
     * [**Topic TBD**](https://www.meetup.com/rust-modern-systems-programming-in-leipzig/events/305741632)
 * 2025-04-15 | London, UK | [Women in Rust](https://www.meetup.com/women-in-rust/events/)
     * [**WIR x WCC: Finding your voice in Tech**](https://www.meetup.com/women-in-rust/events/306774769)
+* 2025-04-23 | London, UK | [London Rust Project Group](https://www.meetup.com/london-rust-project-group/events/)
```

Comment:
Can we add this event in Istanbul? It's on the European side of the river. https://kommunity.com/turkiye-rust-community/events/rust-konf-turkiye-91f7b3a6

Reply:
Added this.